Washington_DC_Shelters/Washington_DC_Shelters.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+class MFB_Scraper(BaseScraper):
+
+    '''For this dataset we need to scrape following columns: FACILITY_NAME(name), PROVIDER(provider), CITY(city),
+    ZIP(zip), WARD(ward), TYPE(type), SUBTYPE(subtype), URL(url), ON_SITE_MEDICAL_CLINIC(medical_clinic),
+    AGES_SERVED(ages_served), HOW_TO_ACCESS(how_to_access),
+    and add an extra serviceSummary column and default it to "Emergency Shelter" for all entries,
+    and add an extra state column and default it to "DC" for all entries.'''
+
+    def grab_data(self):
+        df = pd.read_csv(self.data_url, usecols=self.extract_usecols)
+        df.rename(columns=self.rename_columns, inplace=True)
+        df['medical_clinic'] = np.where(~df['medical_clinic'].isnull(), df['medical_clinic'], 'No')
+        df['serviceSummary'] = self.service_summary
+        df['state'] = 'DC'
+        return df
+
+
+mfb_scraper = MFB_Scraper(
+    source="WashingtonDCServices",
```

Comment:
This value needs to be same as the value for `data_source_collection_name` , so that we know this entry is obtained from which data source. In future we can then have a inner-join on `source` and `data-sources.name` fields.